keycloak_add_users.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+import json
+import requests
+import sys
+import getopt
+
+class updateUserStruct:
+    def __init__(self, id, enabled, username, firstName, lastName, email, attributes):
+        self.id = id
+        self.enabled = enabled
+        self.username = username
+        self.firstName = firstName
+        self.lastName = lastName
+        self.email = email
+        self.attributes = attributes
+
+    def toJson(self):
```

Comment:
Python methods are usual snake_case in name.

Reply:
I'm not sure I agree with the `default` use here. Essentially you're saying if you can't serialize the object, just get its internal dict. As you have set up a precedence of using a `toJson`, which should be named `to_json`, I would imagine you should call `default=lambda o: o.to_json()` perhaps instead. I'm thinking you're hitting an issue because the toJson is a method that isn't JSON serializable. Perhaps it should be specific about what the JSON should contain. Doing it this way means that it could/would expose every attribute on the class.